vms/platformvm/txs/validator.go

```diff
@@ -40,11 +40,6 @@ func (v *Validator) EndTime() time.Time {
 	return time.Unix(int64(v.End), 0)
 }
 
-// Duration is the amount of time that this validator will be in the validator set
-func (v *Validator) Duration() time.Duration {
```

Comment:
Post Durango this is evil. Start and End are there for backward compatiblity reasons, but result has nothing to do with staking duration. Better bury this method